### DISCUSSION TITLE
My build RustDesk cannot be in Windows7 work

### DISCUSSION THREAD

**Discussion by FaQQQ** [rustdesk/rustdesk#952] [Q&A]
![QQ截图20220710102946](https://user-images.githubusercontent.com/38316389/178129279-fcd76825-a774-482a-a37a-c4896a71adde.png)

build ：
![QQ截图20220710105335](https://user-images.githubusercontent.com/38316389/178129473-3a32f0c0-4c56-48c9-9ddb-0039c195b39b.png)

work：
![QQ截图20220710105247](https://user-images.githubusercontent.com/38316389/178129482-01a5152f-d32a-41f3-ab42-ef90de90fdf2.png)

**Comment by Baker9017** [comment]
same problem

**Comment by babalaa** [comment]
also same problem
